Gemfile

```diff
@@ -45,9 +45,11 @@ end
 
 group :test do
   gem 'capybara'
+  gem 'draft_matchers', github: 'jelaniwoods/draft_matchers', branch: 'main'
   gem 'factory_bot_rails'
   gem 'rspec-html-matchers'
   gem 'rspec-rails'
+  gem 'selenium-webdriver', "3.142.7"
```

Comment:
Is this worth adding to `base-rails`, do you think? How about `vanilla-rails`?

Reply:
This is sort of an old version of selenium. If possible, I think it'd be better to add a newer version to base or vanilla rails. The reason I didn't for this project is that there is a breaking change in how you configure the Chrome browser capabilities between version `3.1` and `4` that I wanted to defer until after the tests were finished.